website/docs/r/org_vdc_template.html.markdown

```diff
@@ -0,0 +1,157 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_org_vdc_template"
+sidebar_current: "docs-vcd-resource-org-vdc-template"
+description: |-
+  Provides a resource to create Organization VDC Templates in VMware Cloud Director. This can be used to create, delete, and update a Organization VDC Template.
+---
+
+# vcd\_org\_vdc\_template
+
+Provides a resource to create Organization VDC Templates in VMware Cloud Director. This can be used to create, delete, and update a Organization VDC Template.
+Requires system administrator privileges.
+
+~> Only supports NSX-T network provider
+
+Supported in provider *v3.13+*
+
+## Example Usage
+
+```hcl
+data "vcd_org" "org" {
+  name = "my_org"
+}
+
+data "vcd_provider_vdc" "pvdc1" {
+  name = "nsxTPvdc1"
+}
+
+data "vcd_provider_vdc" "pvdc2" {
+  name = "nsxTPvdc2"
+}
+
+data "vcd_external_network_v2" "ext_net" {
+  name = "nsxt-extnet"
+}
+
+data "vcd_network_pool" "np1" {
+  name = "NSX-T Overlay 1"
+}
+
+resource "vcd_org_vdc_template" "tmpl" {
+  name               = "myTemplate"
+  tenant_name        = "myAwesomeTemplate"
+  description        = "Requires System privileges"
```

Comment:
Nit: would read better overall if this line/param was moved one line above (ordering was like in argument reference docs below).

Reply:
Fixed!